webpack.config.js

```diff
@@ -32,21 +32,14 @@ const sassLoader = {
 		sourceMap: true,
 		sassOptions: {
 			sourceMapContents: false,
-			loadPaths: [
+			includePaths: [
```

Comment:
This breaks netlifiy as the stylesheets can no longer be found

Reply:
Works locally and on main...

Reply:
Ok, on main loader is outdated